README.md

```diff
@@ -45,6 +45,33 @@ Ex. If you are literally running build.sh locally..
 ```bash
 $ docker-machine create --driver virtualbox --virtualbox-memory "4096" dev
 ```
+
+## Connect to Cassandra cluster
+
+When you want to use the docker images not only for development but also for production
```

Comment:
I pushed a commit to the documentation based on your example.

I agree that we can simplify the instructions even more and probably should only keep `docker-compose` as a way to set up a dev environment.